doc/source/api.rst

```diff
@@ -1466,6 +1674,8 @@ Categorical Components
    CategoricalIndex.set_categories
    CategoricalIndex.as_ordered
    CategoricalIndex.as_unordered
+   CategoricalIndex.get_loc
+   CategoricalIndex.map
```

Comment:
Are those needed to include ? (I mean, do they have something categorical specific ?)